components/console/helpers/questionhelper.rst

```diff
@@ -234,6 +234,32 @@ You can also specify if you want to not trim the answer by setting it directly w
         $name = $helper->ask($input, $output, $question);
     }
 
+Accept Multiline Answers
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, the question helper stops reading user input when it receives a newline
```

Comment:
👍 but missing an `versionadded: 5.2` directive here i think

Reply:
Added in my latest push.